build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
 		maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
 	}
 	dependencies {
-		classpath 'com.diffplug.spotless:spotless-plugin-gradle:3.5.1'
+		classpath 'com.diffplug.spotless:spotless-plugin-gradle:3.5.2'
```

Comment:
Needed to bump the version of spotless so that I [could do this](https://github.com/junit-team/junit5/pull/1001/files#diff-70dc7ae1f1905b194a2aaa9505fc4177R10) and not have spotless try to add the header in the wrong place.

See this issue: https://github.com/diffplug/spotless/issues/136
That I resolved here: https://github.com/diffplug/spotless/pull/139